kuma/wiki/utils.py

```diff
@@ -53,6 +58,33 @@ def locale_and_slug_from_path(path, request=None, path_locale=None):
     return (locale, slug, needs_redirect)
 
 
+def get_doc_components_from_url(url, required_locale=None, check_host=True):
```

Comment:
This function requires tests.  ``required_locale`` and ``check_host`` should be removed or fully tested.